### DISCUSSION TITLE
Russian Locale

### DISCUSSION THREAD

**Discussion by ArtM095** [aclap-dev/video-downloadhelper#921] [Language & Translation]
[ru.txt](https://github.com/user-attachments/files/15966433/ru.txt)
all terms, best choise

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
